appnav/src/main/kotlin/io/element/android/appnav/room/joined/JoinedRoomLoadedFlowNode.kt

```diff
@@ -104,11 +104,6 @@ class JoinedRoomLoadedFlowNode @AssistedInject constructor(
                     inputs.room.subscribeToSync()
                 }
             },
-            onPause = {
-                appCoroutineScope.launch {
-                    inputs.room.unsubscribeFromSync()
```

Comment:
`unsubscribeFromSync` does not exist anymore?

Reply:
No, it was removed from the SDK. According to Ivan it should be alright (apparently, unsubscribing is more expensive performance-wise for the server), and I didn't experience any performance issues either.

Reply:
OK, thanks!